tests/integration/courses.test.js

```diff
@@ -90,12 +113,27 @@ describe('COURSES ROUTES - POST /courses', () => {
       expect(response.statusCode).toBe(403);
     });
 
+    it('should return 403 if invalid salesRepresentative', async () => {
```

Comment:
je ne comprends pas ton it
- le referent Compani me parait bon puisque c'est un admin vendeur
- tu testes une 400 a la fin de ton test 

ce serait plutot `if invalid type` non ?